src/Grpc/test/InteropTests/InteropTests.cs

```diff
@@ -27,6 +27,7 @@ public InteropTests(ITestOutputHelper output)
         }
 
         [Theory]
+        [Repeat]
```

Comment:
Runs 10 times which should take an extra 20 mins.

Reply:
You'll run the PR build a few times with this and then remove it before checking in, right?

Reply:
That's the plan.